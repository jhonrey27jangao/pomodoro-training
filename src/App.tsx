import React from "react";
import Main from "./pages";
const Portfolio: React.FC = () => {
  return <Main />;
};

Portfolio.displayName = "Portfolio";
export default Portfolio;
