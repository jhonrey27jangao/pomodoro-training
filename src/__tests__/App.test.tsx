import React from "react";
import { render } from "@testing-library/react";
import App from "../App";

describe("Test for App Component", () => {
  it("should render App", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
