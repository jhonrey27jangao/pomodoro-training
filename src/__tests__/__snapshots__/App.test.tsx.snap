// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for App Component should render App 1`] = `
<div>
  <div
    id="main"
  >
    <div
      class="sc-fznJRM kcNyfq"
    >
      <div
        class="sc-fzoiQi eJBvFQ"
      >
        <a
          href="sadad"
        >
          Ricardo.
        </a>
      </div>
      <div
        class="sc-fznWqX eLDCzX"
      >
        <ul
          class="sideBar-nav"
        >
          <a
            href="/"
          >
            <li>
              Home
            </li>
          </a>
          <a
            href="/pomodoro"
          >
            <li>
              Pomodoro
            </li>
          </a>
          <a
            href="/covid-tracker"
          >
            <li>
              Covid Tracker
            </li>
          </a>
        </ul>
        <ul
          class="sideBar-social"
        >
          <li
            class="ss-fb"
          >
            <a
              href="https://facebook.com"
            >
              <span>
                Facebook
              </span>
            </a>
          </li>
          <li
            class="ss-insta"
          >
            <a
              href="https://instagram.com"
            >
              <span>
                Instagram
              </span>
            </a>
          </li>
          <li
            class="ss-dribble"
          >
            <a
              href="https://dribble.com"
            >
              <span>
                Gmail
              </span>
            </a>
          </li>
          <li
            class="ss-twitter"
          >
            <a
              href="https://twitter.com"
            >
              <span>
                Twitter
              </span>
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      class="sc-fzqARJ kxkqau"
      title="Covid Tracker:"
    >
      <h1
        class="title-page"
      >
        Covid Tracker:
      </h1>
      <div />
    </div>
  </div>
</div>
`;
